docs/tutorials/advanced/strategy_transformers.rst

```diff
@@ -100,6 +96,26 @@ The library includes the following transformers:
     >>> from axelrod.strategy_transformers import TrackHistoryTransformer
     >>> player = TrackHistoryTransformer(axelrod.Random)()
 
+
+Composing Transformers
+----------------------
+
+Transformers can be composed to form new composers, in two ways. You can
+simply chain together multiple transformers::
+
+    cls1 = FinalTransformer([D,D])(InitialTransformer([D,D])(axelrod.Cooperator))
+    p1 = cls1()
+
+This defines a strategy that cooperates except on the first two and last two
+rounds. Alternatively, you can make a new class using
+:code:`compose_transformers`::
+
+    cls1 = compose_transformers(FinalTransformer([D, D]),
+                                InitialTransformer([D, D]))
+    p1 = cls1(axelrod.Cooperator)()
```

Comment:
Could these be doctested `>>>`, I know there is no output but say that at some point the input changes etc and typing this would give an error?

Same for quite a few here (sorry should have raised this in previous PR).